ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/apps/sync-new/ManualSync.tsx

```diff
@@ -59,27 +59,30 @@ export default function ManualSync({ appsURL }: Props) {
     <>
       <div className="border-b border-slate-200 p-8">
         <p>
-          Inngest allows you to host your app code on any platform then invokes your functions via
-          HTTP. To deploy functions to Inngest, all you need to do is tell Inngest where to find
-          them!
+          To integrate your code hosted on another platform with Inngest, you need to inform Inngest
+          about the location of your app and functions.
         </p>
         <br />
         <p>
-          Since your code is hosted on another platform, you need to register where your functions
-          are hosted with Inngest. For example, if you set up the serve handler (
+          For example, imagine that your <Code>serve()</Code> handler (
           <Link
-            className={defaultLinkStyles}
+            showIcon={false}
             href="https://www.inngest.com/docs/reference/serve#how-the-serve-api-handler-works"
           >
             see docs
           </Link>
-          ) at /api/inngest, and your domain is https://myapp.com, you&apos;ll need to inform
-          Inngest that your app is hosted at https://myapp.com/api/inngest.
+          ) is located at /api/inngest, and your domain is myapp.com. In this scenario, you&apos;ll
+          need to inform Inngest that your apps and functions are hosted at
+          https://myapp.com/api/inngest.
         </p>
         <br />
         <p>
-          After you&apos;ve set up the serve API and deployed your application, enter the URL of
-          your application&apos;s serve endpoint to register your functions with Inngest.
+          After you&apos;ve set up the serve API and deployed your code,{' '}
+          <span className="font-semibold">
+            enter the URL of your project&apos;s serve endpoint to sync your app with Inngest
+          </span>
+          . Verify that you assigned the signing key below to the right{' '}
```

Comment:
```suggestion
          . Verify that you assigned the signing key below to the correct{' '}
```

Super small nit: I read "below to the right" as "bottom right"